class exc2
{
    public static void main(String args[])
    {
        int a,d;
        try
        {
            d=0;
            a=42/d;
            System.out.println("in try block");
        }
        catch(ArithmeticException e)
        {
            System.out.println("Division by zero");
        }
    }
}
